portfolio/src/main/webapp/script.js

```diff
@@ -39,9 +39,24 @@ function toggleDetailsButton() {
   }
 }
 
-function getHelloMessage() {
-    fetch('/data').then(response => response.text()).then((message) => {
-        document.getElementById('message-container').innerText = message;
+// gets the data from server sides
```

Comment:
We usually starts comments with upper case.

Reply:
done